examples/isomorphic-flux-chat/js/main.js

```diff
@@ -0,0 +1,25 @@
+var mockData = require('./mock-data')
+var Nuclear = require('nuclear-js')
+var reactor = new Nuclear.Reactor({
+  debug: true,
+})
+window.reactor = reactor
+var Chat = require('./modules/chat')
+
+var ChatApp = require('./components/ChatApp.react')
+ChatApp = Nuclear.provideReactor(ChatApp)
```

Comment:
It's definitely a plus to have the `ChatApp` require the `reactor` context, of course, which this pattern does...